lib/bamboo/adapters/smtp_adapter.ex

```diff
@@ -161,6 +168,45 @@ defmodule Bamboo.SMTPAdapter do
     |> add_smtp_line(text_body)
   end
 
+  defp add_multipart_mixed_split(body, delimiter), do: "#{body}\r\n--#{delimiter}\r\n"
+
+  defp add_attachment_header(body, attachment) do
+    << random :: size(32) >> = :crypto.strong_rand_bytes(4)
+    "#{body}Content-Type: attachment.content_type; name=\"#{attachment.filename}\"\r\n" <>
```

Comment:
Sorry if my comment wasn't clear enough  :)

I meant we should use the value of the `attachment.content_type` variable instead of the literal string `"attachement.content_type"`:

```diff
- |> add_smtp_line("Content-Type: attachment.content_type; name=\"#{attachment.filename}\"")
+ |> add_smtp_line("Content-Type: #{attachment.content_type}; name=\"#{attachment.filename}\"")
```